File: src/net/eduard/economy/command/EconomyHelpCommand.java

package net.eduard.economy.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;

public class EconomyHelpCommand extends CommandManager {

	public EconomyHelpCommand() {
		super("help","ajuda","?");
		setDescription("Ver como funciona os comandos");
	}
	@Override
	public boolean onCommand(CommandSender sender, Command command,
			String label, String[] args) {
		for (CommandManager cmd : getParent().getCommands().values()) {
			sender.sendMessage("§7"+cmd.getUsage()+" §8- §f"+cmd.getDescription());
		}
		return true;
	}

}

File: src/main/java/net/eduard/economy/command/EconomyHelpCommand.kt
package net.eduard.economy.command

import net.eduard.api.lib.manager.CommandManager
import org.bukkit.command.CommandSender

class EconomyHelpCommand : CommandManager("help", "ajuda", "?") {
    override fun command(sender: CommandSender, args: Array<String>) {
        for (cmd in parent!!.subCommands.values) {
            sender.sendMessage("§7" + cmd.usage + " §8- §f" + cmd.description)
        }
    }

    init {
        description = "Ver como funciona os comandos"
    }
}